backend/src/main/java/com/funeat/recipe/dto/RecipeCreateRequest.java

```diff
@@ -1,11 +1,21 @@
 package com.funeat.recipe.dto;
 
 import java.util.List;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 
 public class RecipeCreateRequest {
 
+    @NotBlank(message = "레시피 이름을 확인해 주세요")
     private final String title;
+
+    @NotNull(message = "상품 ID 목록을 확인해 주세요")
+    @Size(min = 1, message = "적어도 1개의 상품 ID가 필요합니다")
     private final List<Long> productIds;
+
+    @NotBlank(message = "레시피 내용을 확인해 주세요")
+    @Size(max = 500, message = "리뷰 내용은 최대 500자까지 입력 가능합니다")
```

Comment:
```suggestion
    @Size(max = 500, message = "레시피 내용은 최대 500자까지 입력 가능합니다")
```
리뷰 -> 레시피가 더 어울릴 것 같아요!